server/tso/allocator_manager.go

```diff
@@ -47,6 +47,7 @@ const (
 	patrolStep                  = 1 * time.Second
 	defaultAllocatorLeaderLease = 3
 	leaderTickInterval          = 50 * time.Millisecond
+	localTSOAllocatorEtcdPrefix = "local-tso-allocator"
```

Comment:
Do we have a compatibility issue?

Reply:
Since the Local TSO has not been released yet, I think it's ok to break the backward compatibility.

Reply:
It's better to be shorter, like "lta"?

Reply:
BTW, should we use `-` or `_`?

Reply:
I think `_` is better since we already have keys like `alloc_id`.